hudai/client.py

```diff
@@ -60,6 +60,7 @@ def __init__(self,
         self.relevant_article_collateral = RelevantArticleCollateralResource(self._http_client)
         self.relevant_articles = RelevantArticlesResource(self._http_client)
         self.stock_alerts = StockAlertResource(self._http_client)
+        self.sources = SourceResource(self._http_client)
```

Comment:
do you export source resource correctly so that it is available here?

Reply:
You got me